src/darwin/Framework/CHIP/MTRDefines.h

```diff
@@ -37,13 +37,36 @@
 
 #pragma mark - Deprecation macros (can be overriden via build system)
 
+/**
+ * MTR_SKIP_AVAILABILITY_ANNOTATIONS can be used to turn off availability
```

Comment:
Could call this `MTR_NO_AVAILABILITY` if we wanted a shorter name.

Reply:
Yep, can do.